docs/permissions.md

```diff
@@ -146,6 +146,10 @@ Additionally, we need to grant Pub/Sub publisher permission to GCS service agent
    gcloud projects add-iam-policy-binding PROJECT_ID --member=serviceAccount:<GCS_SERVICE_ACCOUNT_ID> --role=roles/pubsub.publisher
   ```
 
+### Cloud Monitoring
+
+Follow [this guide](https://cloud.google.com/monitoring/access-control) to enable Cloud Monitoring permissions.
+
```

Comment:
Lets' be more specific, list the specific roles and permissions required to allow HB to create a dashboard on the user's behalf and the link to the steps required to add permissions to a given IAM user/service account.

Example of listing permissions - 
https://googlecloudplatform.github.io/spanner-migration-tool/permissions.html#spanner